SourceX/miniwin/misc.cpp

```diff
@@ -530,6 +560,17 @@ int ReleaseDC(HWND hWnd, HDC hDC)
 
 int GetDeviceCaps(HDC hdc, int index)
 {
+	#ifdef USE_SDL1
+	const SDL_VideoInfo *info = SDL_GetVideoInfo();

```

Comment:
Would it be possible to wrap this in an implementation of `SDL_GetCurrentDisplayMode`?

Reply:
Done